platforms/common.mk

```diff
@@ -2,6 +2,8 @@ PLATFORM_COMMON_DIR = $(PLATFORM_PATH)/$(PLATFORM_KEY)
 
 TMK_COMMON_SRC +=	\
 	$(PLATFORM_PATH)/suspend.c \
+	$(PLATFORM_PATH)/synchronization_util_common.c \
+	$(PLATFORM_PATH)/timer_common.c \
```

Comment:
We shouldnt introduce a `_common` suffix here, when no other files follow that pattern.

Reply:
Could having multiple `timer.c` and `synchronization_util.c` files in the tree cause problems somewhere?  Apparently in this particular case files are specified using the full path, but some other parts of the build system rely on `VPATH`.

Reply:
If it were an issue, then it would be an issue with other files. I would expect any solution and changing of direction, to be within a separate PR and not this one.

Reply:
OK, renamed the files (apparently `suspend.c` is already at both levels and works fine).